pointsource_imaging/MIRI_Aperture_Photometry_solution.ipynb

```diff
@@ -161,14 +161,27 @@
     "### 2.2<font color='white'>-</font>Plot configuration<a class=\"anchor\" id=\"plot\"></a> ###"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ax.set_box_aspect(1.75)
</pre><p>Did you mean <code>ax.set_aspect(1.75)</code> here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='118'/>

Reply:
I need a rectangle figure. `ax.set_box_aspect` did the jobs, but I changed with your suggestion, fine-tuning the value a bit: `ax.set_aspect(0.35)`